docs/en/how-to/deploy_toolbox.md

```diff
@@ -197,3 +205,26 @@ func main() {
 
 
 Now, you can use this client to connect to the deployed Cloud Run instance!
+
+## Troubleshooting
+
+* **Deployment Fails with "Container failed to start":** This is almost always
+    caused by a port mismatch. Ensure your container's `--port` argument is set to
+    `8080` to match the `$PORT` environment variable provided by Cloud Run.
+
+* **Permission Denied Errors (401 or 403):** If you receive permission errors when accessing secrets or invoking the Cloud Run
```

Comment:
We should split secrets permission errors with Cloud Run auth errors. The 401/403 errors typically make me think about the request to the Cloud Run instance versus internal logs that are showing no access to the secret.

Reply:
Sure, got your point!